front/src/components/nft-card-information/NftCardInformationStyle.ts

```diff
@@ -0,0 +1,20 @@
+import { createStyles } from "@mantine/core";
+
+export const useStyles = createStyles(() => ({
+    textBold: {
+        fontWeight: 400,
+    },
+
+    listSize: {
+        margin: "20px 0 20px;",
```

Comment:
pls use theme variables (with a shorthand property)

Reply:
Its correct write like this?

margin: [`20px 0 ${theme.spacing.xl}px`],

Reply:
@nastyakuzusha plz answer